CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Changes by Version
 ==================
 
+1.22.2 (2021-06-01)
+-------------------
+
+* Add preserve unknown fields annotation to FreeForm and Options fields ([#1435](https://github.com/jaegertracing/jaeger-operator/pull/1435), [@rubenvp8510](https://github.com/rubenvp8510))
+* Update opentelemetry to 0.20.0 ([#1446](https://github.com/jaegertracing/jaeger-operator/pull/1446), [@rubenvp8510](https://github.com/rubenvp8510))
```

Comment:
Remove this one, it might confuse users of the operator.

Reply:
Removed